crates/flipperzero/src/lib.rs

```diff
@@ -1,10 +1,14 @@
 //! High-level bindings for the Flipper Zero.
-
+//!
+//! # Features
+#![cfg_attr(feature = "document-features", doc = document_features::document_features!())]
 #![no_std]
 #![cfg_attr(test, no_main)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
 #![deny(rustdoc::broken_intra_doc_links)]
 
-#[cfg(any(feature = "alloc", docsrs))]
+#[cfg(feature = "alloc")]
```

Comment:
This change breaks intra-doc links. Revert it.
```suggestion
#[cfg(any(feature = "alloc", docsrs))]
```

Reply:
I think that it would be better to enable `--all-features` for docs, but I will bring this back as for mow.

Reply:
Done